Documentation/project-docs/standard-platform.md

```diff
@@ -41,7 +41,7 @@ In general, class libraries which target a lower .NET Platform Standard version
 
 | Target Platform Name | Alias |  |  |  |  |  |
 | :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Platform Standard | dotnet | 5.1 | 5.2 | 5.3 | 5.4 | 5.5 |
+|.NET Platform Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 |
```

Comment:
>Did we consider making this shorter? Perhaps netstd? 

I would vote we do not go with 'netstd' as a choice. The 'std' suffix might be misinterpreted.

What about 'netplat' (i.e. .NET Platform)?